etl/churn.kp/knowledge.md

```diff
@@ -422,45 +413,6 @@ def get_newest_per_client(df):
     )
     return selectable_by_client.filter(selectable_by_client['row_number'] == 1)
 
-
-def upload_to_s3(df, churn_outfile):
-    client = boto3.client('s3', 'us-west-2')
-    transfer = S3Transfer(client)
-    churn_bucket = "net-mozaws-prod-us-west-2-pipeline-analysis"
-    churn_filepath = "amiyaguchi/churn"
-
-    print "{}: Writing output to {}".format(_datetime.utcnow(), churn_outfile)
-
-    # Write the file out as gzipped csv
-    with gzip.open(churn_outfile, 'wb') as fout:
-        fout.write(",".join(df.columns) + "\n")
-        print "{}: Wrote header to {}".format(_datetime.utcnow(), churn_outfile)
-        records = df.rdd.collect()
-        for r in records:
-            try:
-                fout.write(csv(r))
-                fout.write("\n")
-            except UnicodeEncodeError as e:
-                print("{}: Error writing line: {} // {}"
-                      .format(_datetime.utcnow(), e, r))
-        print "{}: finished writing lines".format(_datetime.utcnow())
-
-    # Now upload it to S3:
-    churn_s3 = "{}/{}".format(churn_filepath, churn_outfile)
-    transfer.upload_file(churn_outfile, churn_bucket, churn_s3,
-                         extra_args={'ACL': 'bucket-owner-full-control'})
-
-    # TODO: Re-enable uploading to the dashboard when cutover happends
-    ENABLE_UPLOAD_DASHBOARD = False
-
-    if ENABLE_UPLOAD_DASHBOARD:
-        # Also upload it to the dashboard:
-        # Update the dashboard file
-        dash_bucket = "net-mozaws-prod-metrics-data"
-        dash_s3_name = "telemetry-churn/{}".format(churn_outfile)
-        transfer.upload_file(churn_outfile, dash_bucket, dash_s3_name,
-                             extra_args={'ACL': 'bucket-owner-full-control'})
-
     
 def compute_churn_week(df, week_start, bucket, prefix, enable_upload_csv=False):
```

Comment:
Please remove the enable_upload_csv parameter since it's no longer used. Also update the docstring for this function and the call sites to reflect this change.